apps/core-app/src/pages/Dao.tsx

```diff
@@ -2,12 +2,34 @@ import { useParams, Outlet } from 'react-router-dom';
 import { HausLayout, useHausConnect } from '@daohaus/daohaus-connect-feature';
 import { useDao } from '@daohaus/dao-context';
 import { TXBuilder } from '@daohaus/tx-builder-feature';
+import { AppSwitcher } from '@daohaus/ui';
 
 export function Dao() {
   const { daochain, daoid } = useParams();
   const { provider } = useHausConnect();
   const { dao } = useDao();
 
+  const apps = {
+    trigger: {
+      name: 'Core App',
+      url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
```

Comment:
can you check that this is the right link @skuhlmann ?

Reply:
fyi - this url is not active, its the trigger for the dropdown -- it just cant be blank

Reply:
@brossetti1 lgtm